src/screens/Profiles/UserLibrary/components/UserLibraryEditScreen/styles.js

```diff
@@ -7,6 +7,25 @@ export const styles = StyleSheet.create({
     flex: 1,
     backgroundColor: colors.white,
   },
+  loadingContainer: {
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    bottom: 0,
+    right: 0,
+    backgroundColor: 'rgba(0,0,0,0.7)',
+    alignItems: 'center',
+    justifyContent: 'center',
+    zIndex: 666,
```

Comment:
Should we export these indexes to layer => into a constants file.

Just a thought.